src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,64 @@
+.page__clock {
+  position: relative;
+}
+
+.clock {
+  width: $clock-size;
+  height: $clock-size;
+  box-shadow: 1px 1px 20px #1b8525;
+  border-radius: $circle;
+  background-image: url(../images/clock-1302406_1280.webp);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+
+  &__image {
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    transform: translate(-50%, 0);
+    width: $image-size;
+    height: $image-size;
+  }
+
+  &__min {
+    @include position;
+    @include transform;
+    width: 6px;
+    height: 180px;
+    border-radius: 10px;
+    background-color: #000;
+    animation: move 180s steps(12) infinite forwards;
+    transform-origin: top left;
+  }
+
+  &__sec {
+    @include position;
+    @include transform;
+    width: 6px;
+    height: 140px;
+    border-radius: 10px;
+    background-color: $red-color;
+    animation: move 15s steps(60) infinite forwards;
+    transform-origin: top left;
+  }
+
+  @keyframes move {
+    from {
+      transform: rotate(180deg);
+    }
+
+    to {
+      transform: rotate(540deg);
```

Comment:
Better to set from 0 to 360 deg and set starting position so that arrow points up at 0 degrees

Reply:
I don't know how I can do it. I did use rotate: 180deg; for min and sec hand... I think it's incorrect, because during animation active state everything looks right. But when I stop animation in dev tools - hands position go down and they go go beyond the parent element.